.github/workflows/locale_po_to_json.yaml

```diff
@@ -0,0 +1,68 @@
+name: "Locale PO to JSON"
+
+on:
+  workflow_dispatch:
```

Comment:
For now, no cron job... just run it ad hoc

Reply:
That's perfect

Reply:
Do you want to add `repository_dispatch` now to set up the next set of PRs, or wait till then?

Reply:
I can add it later when we figure out how we want to run it automatically